src/Framework.Cqrs/Mediator/V1/IMediator.cs

```diff
@@ -6,7 +6,7 @@
 
```

Comment:
This class can be deleted. We don't need to store deprecated classes  :). 

My fault for still being here.

Reply:
Fixed in 08b302192c219268a64aede2f07e83f05ebb3211